.github/workflows/autoclose.yml

```diff
@@ -10,14 +10,14 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@v8
         with:
           only-labels: "Status: Needs Author Input"
           days-before-issue-stale: 30
           days-before-issue-close: 90
           stale-issue-label: "Status: Stale"
-          stale-issue-message: "This issue is stale because the author @${{ github.event.issue.user.login }} has not responded for 30 days."
-          close-issue-message: "This issue was closed because the author @${{ github.event.issue.user.login }} has not responded for 90 days since the issue was marked as stale."
+          stale-issue-message: "Hey there @${{ github.event.pull_request.user.login }} , We wanted to express our appreciation for your initial contribution to this issue. However, it has been marked as stale since we haven't received any response from you in the past 30 days. If you're still available, we would greatly appreciate it if you could provide answers to any open questions and/or share the requested feedback/input. Thank you for your consideration, and we hope to hear from you soon!"
```

Comment:
```suggestion
          stale-issue-message: "Hey there @${{ github.event.pull_request.user.login }} , we want to express our appreciation for your initial contribution to this issue. However, we now mark it as stale since we haven't received any response from you in the past 30 days. If you're still available, we would greatly appreciate it if you could provide answers to any open questions and/or share the requested feedback/input. Thank you for your consideration, we hope to hear from you soon!"
```